torchao/dtypes/uintx/tensor_core_tiled_layout.py

```diff
@@ -383,3 +393,251 @@ def get_plain(self) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
 
     def get_layout(self) -> Layout:
         return self._layout
+
+
+@dataclass(frozen=True)
```

Comment:
oh sorry missed this one, it should have a separate file since it's a different layout, cc @yanbing-j can you help move this to a separate file under the same directly? (int4_cpu_layout.py)

Reply:
@jerryzh168  Okay, here is the PR https://github.com/pytorch/ao/pull/1419.